package.json

```diff
@@ -2,10 +2,7 @@
   "name": "twitter-media",
   "version": "0.1.0",
```

Comment:
It should only be a patch release (no API changes). Bumping the version should be done when you publish rather than as part of this PR:

```
$ npm version patch
$ git add package.json
$ git commit -m "Version 0.1.1"
$ git push && git push --tags
$ npm publish
```